packages/xo-server/src/xo-mixins/remotes.js

```diff
@@ -78,17 +85,21 @@ export default class {
   }
 
   async getAllRemotes () {
-    return this._remotes.get()
+    return (await this._remotes.get()).map(_ => obfuscateRemote(_))
```

Comment:
Don't you (or will you) ever need to get all unobfuscated remotes?

Reply:
Public methods (without `_` prefix) should not contain sensitive data, if necessary I will add `_getAllRemote` similarly to `_getRemote`.